src/main/java/org/gridsuite/modification/server/modifications/TwoWindingsTransformerModification.java

```diff
@@ -381,8 +381,8 @@ private void addRatioTapChangersToTwoWindingsTransformer(Network network, TwoWin
 
     private boolean ratioTapChangerModified(RatioTapChangerModificationInfos ratioTapChangerModificationInfos) {
         return ratioTapChangerModificationInfos != null && (
-                ratioTapChangerModificationInfos.getLoadTapChangingCapabilities() != null
-                && ratioTapChangerModificationInfos.getLoadTapChangingCapabilities().getValue() != null
+                ratioTapChangerModificationInfos.getHasLoadTapChangingCapabilities() != null
+                && ratioTapChangerModificationInfos.getHasLoadTapChangingCapabilities().getValue() != null
```

Comment:
This verification is used Two times. Could you extract it  to a single method and use it here and above.